File: accsaber-database/src/main/java/de/ixsen/accsaber/database/repositories/model/CategoryPerformanceRepository.java
package de.ixsen.accsaber.database.repositories.model;

import de.ixsen.accsaber.database.model.PlayerCategoryStats;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface CategoryPerformanceRepository extends JpaRepository<PlayerCategoryStats, Long> {

}

File: accsaber-database/src/main/kotlin/de/ixsen/accsaber/database/repositories/model/CategoryPerformanceRepository.kt
package de.ixsen.accsaber.database.repositories.model

import de.ixsen.accsaber.database.model.PlayerCategoryStats
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface CategoryPerformanceRepository : JpaRepository<PlayerCategoryStats, Long>